src/chains.ts

```diff
@@ -36,18 +36,23 @@ export const alephzeroTestnet: SubstrateChain = {
   faucetUrls: ['https://faucet.test.azero.dev'],
 }
 
-export const rococo: SubstrateChain = {
-  network: 'rococo',
-  name: 'Rococo Contracts Testnet',
+export const contracts: SubstrateChain = {
```

Comment:
Do you really think `contracts` is descriptive enough? I personally wouldn't know what's going on w/o looking it up! Maybe rather a mix, wdyt of `rococoContracts`?

Reply:
> Writing this, maybe it would be a good idea to add a general check with the pjs api to check if the pallet-contracts is even available on the given chain. If not throw an Error about it.

Just FYI, I'm doing this in an example already: https://github.com/scio-labs/use-inkathon/blob/70e0af6c1e97795f27ff7b371d389ecbb8ea2f60/examples/react-ts/src/components/ConnectionStatus.tsx#L11-L16. But I think I think it's not mandatory for the library (haven't heard about one having troubles with this as most peps know which chain they gonna build on or just use the chains which are provided by the library).